tests/lib/rules/space-before-keywords.js

```diff
@@ -218,25 +218,6 @@ ruleTester.run("space-before-keywords", rule, {
             code: "; class Bar extends Foo.Baz {}",
             ecmaFeatures: { classes: true },
             options: never
-        },
-        // Super
-        {
```

Comment:
The valid ones. In general, we want to avoid removing valid test cases for rules to ensure backwards compatibility. Can you add these back?